cmd/helm/version.go

```diff
@@ -89,6 +90,13 @@ func newVersionCmd(c helm.Interface, out io.Writer) *cobra.Command {
 }
 
 func (v *versionCmd) run() error {
+	if settings.Debug {
```

Comment:
hey! Sorry about the long wait on the review here.

Would you mind moving this after line 106? If we're making a server-side call to check the kubernetes version, we will want to respect the `--client-only` flag.